Godeps/_workspace/src/k8s.io/kubernetes/test/e2e/kubectl.go

```diff
@@ -755,7 +755,7 @@ var _ = Describe("Kubectl client", func() {
 		var rcPath string
 		var nsFlag string
 		containerName := "redis-master"
-		BeforeEach(func() {
+		JustBeforeEach(func() {
```

Comment:
updated the other `BeforeEach` items in this file that were creating pods.  After a sweep it looked like the other cases where we might change `BeforeEach` to `JustBeforeEach` were all in tests that are marked as skip so I left them alone for now.